lms/djangoapps/verify_student/migrations/0009_auto__change_softwaresecurephotoverification_window_id_default_none.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import connection
+
+class Migration(SchemaMigration):
+
+    def forwards(self):
+
+        # Changing field 'SoftwareSecurePhotoVerification.window'. Setting its default value to None
+        cursor = connection.cursor()
+        cursor.execute('ALTER TABLE verify_student_softwaresecurephotoverification CHANGE `window_id` `window_id` int(11) DEFAULT NULL;')
+
+    def backwards(self, orm):
+
+        # Changing field 'SoftwareSecurePhotoVerification.window'
+        db.alter_column('verify_student_softwaresecurephotoverification', 'window_id', self.gf('django.db.models.fields.related.ForeignKey')(null=True,  to=orm['reverification.MidcourseReverificationWindow']))
```

Comment:
This looks auto-generated, and I'm not sure that it reverses the effects of the forward migration.  Please test this by running the migration forwards and backwards in devstack.  (You should be able to use `./manage.py lms migrate 0008 credit --settings=devstack` to migrate backwards)